src/src/com/microsoft/aad/adal/StorageHelper.java

```diff
@@ -127,7 +127,7 @@
 
     private static String sBlobVersion;
```

Comment:
k, so the key and macKey is no longer static, they belong to the storagehelper instance. If one instance fail to get the key and macKey, then I don't even have anything to encrypt, right? Generally, I don't want us to have the static class variable in the class, it's dangerous here. 

I'll stop arguing here, i do agree this should also be non-static. :)